Source/WebCore/accessibility/AccessibilityObject.cpp

```diff
@@ -179,13 +179,28 @@ String AccessibilityObject::debugDescriptionInternal(bool verbose, std::optional
     return result.toString();
 }
 
-void AccessibilityObject::detachRemoteParts(AccessibilityDetachmentType detachmentType)
+String AccessibilityObject::extraDebugInfo() const
 {
-    // Menu close events need to notify the platform. No element is used in the notification because it's a destruction event.
-    if (detachmentType == AccessibilityDetachmentType::ElementDestroyed && role() == AccessibilityRole::Menu) {
-        if (auto* cache = axObjectCache())
-            cache->postNotification(nullptr, cache->document(), AXNotification::MenuClosed);
+    if (isContinuation())
```

Comment:
Should this be on AccessibilityNodeObject? I feel like (for now at least) AccessibilityObject is supposed to be for common code that doesn't assume you have a node or a renderer, right?